TYPO3.Media/Classes/TYPO3/Media/Controller/AssetController.php

```diff
@@ -607,7 +607,7 @@ protected function maximumFileUploadSize()
     public function addFlashMessage($messageBody, $messageTitle = '', $severity = Message::SEVERITY_OK, array $messageArguments = array(), $messageCode = null)
     {
         if (is_string($messageBody)) {
-            $messageBody = $this->translator->translateById($messageBody, $messageArguments, null, null, 'Main', 'TYPO3.Media') ?: $messageBody;
+            $translatedMessageBody = $this->translator->translateById($messageBody, $messageArguments, null, null, 'Main', 'TYPO3.Media') ?: $messageBody;
```

Comment:
… also the ternary should have been removed, this way it creates a useless no-op.